lib/hanami/cli/registry.rb

```diff
@@ -158,6 +270,28 @@ def command(command_name)
         end
       end
 
+      # @since 0.2.0
+      # @api private
+      #
+      # rubocop:disable Metrics/MethodLength
+      def _callback(callback, blk)
+        return blk if blk.respond_to?(:to_proc)
+
+        case callback
+        when ->(c) { c.respond_to?(:call) }
+          callback.method(:call)
+        when Class
```

Comment:
are you sure that we want to work with class callbacks?

Reply:
@davydovanton nope, it **accepts** a class as an argument and turns it into an object (see the next line). Is it clear the intent?

Reply:
yep, I see 👍 
I mean other, do we want to allow `Callbacks::World` and `Callbacks::World.new` instead only instead objects objects?

Reply:
Because the syntax that uses classes is consistent with the current command registration syntax:

```ruby
# Command registration uses a class
Foo::CLI::Commands.register "hello",  Foo::Webpack::CLI::Hello

# Callback registration can use a class
Foo::CLI::Commands.before("callbacks", Callbacks::BeforeClass)
```